crates/types/src/rpc/transaction_receipt.rs

```diff
@@ -81,39 +100,3 @@ pub struct MessageToL1 {
     pub to_address: EthAddress,
     pub payload: L2ToL1Payload,
 }
-
-#[cfg(test)]
-mod tests {
```

Comment:
Why are these tests deleted? Did they get replaced somewhere else?

Reply:
because the transaction receipt have different format and its no longer accurate. the spec_reader is not taking care of it. Issue #248 is for that